bin/install-prerelease.sh

```diff
@@ -1,8 +1,18 @@
 #!/bin/bash
 
-PRERELEASE_NODE_VERSION=$1
+CONFIGURED_PRERELEASE_NODE_VERSION=$1
 
-echo "Installing Node.js prerelease version $PRERELEASE_NODE_VERSION..."
+fetch_latest_prerelease_node_version() {
+  local latest_version
+  
+  latest_version=$(curl -sL https://nodejs.org/dist/index.json | jq -r '[.[]][0].version')
+  latest_version=${latest_version#v}
+  IFS='.' read -r major minor patch <<< "$latest_version"
+  major=$((major + 1))
+  echo "${major}.0.0-rc"
```

Comment:
```suggestion
  echo "${major}"
```
It seems there will be an issue with how the prerelease version is being constructed as `${major}.0.0-rc`, which results in it being set to `24.0.0-rc`. This approach prevents the correct installation of the nightly release, as the prerelease version should simply be `24`.

Reply:
Yes thanks! updated